libraries/chain/webassembly/console.cpp

```diff
@@ -39,7 +39,7 @@ namespace eosio { namespace chain { namespace webassembly {
       });
    }
 
-   void interface::printi128(const __int128& val) {
+   void interface::printi128(legacy_ptr<const __int128> val) {
 		predicated_print(context,
       [&]() {
 			bool is_negative = (val < 0);
```

Comment:
The fact that no code changes were needed here despite the change in function signature is a little concerning. I don't like that a `reference_proxy` type can implicitly convert to both a pointer and a reference. This is why I suggest removing the implicit conversion to references from `reference_proxy` (which I suggest renaming to `proxy_pointer`).